docs/operational_guide/replication_and_deployment_in_zones.md

```diff
@@ -0,0 +1,57 @@
+
+Here's what a deployment in a single zone looks like, in this example the cluster has 3 shards that are spread over 4 racks:
+
+![Replication Single Zone](replication_single_zone.png)
+
+Typically deployments have many more than 3 shards, but this is a simple example that depicts how losing a single rack still maintains availability of two of the three replicas of each shard.
+
+## Deployment across multiple availability zones in a region
+
+For deployment in a region, we recommend you set the `isolationGroup` to the name of the availability zone a host resides.
+
+In this configuration hosts are distributed shards of data that will not be placed more than once in the same availability zone.
+
+Here's what a deployment in a region looks like, in this example the cluster has 4 shards that are spread over 3 availability zones:
+
+![Replication Region](replication_region.png)
+
+Typically deployments have many more than 4 shards, but this is a simple example that depicts how losing a single availability zone still maintains availability of two of the three replicas of each shard.
+
+<!-- **Note:** Placement of shards in a replica do not mirror other replicas and shards are placed in a way to best load balance shards across available hosts in a single isolation group. This could mean that hosts in an availability zones that have less hosts will do more work than those in an availability zone with more hosts. -->
+
+## Deployment across multiple regions
+
+For deployment across regions, we recommend you set the `isolationGroup` to the name of the region a host resides.
+
+As mentioned before you may experience higher ingestion latency and cross region bandwidth costs for a cluster that spans regions. You may also want to use a replication factor of five for such a multi-region cluster to survive multiple regions suffering an outage, or be able to still upgrade hosts while a region is down.
+
+In this configuration hosts are distributed shards of data that will not be placed more than once in the same region.
+
+Here's what a deployment across regions looks like, in this example the cluster has 4 shards that are spread over 5 regions:
+
+![Replication Global](replication_global.png)
+
+Typically deployments have many more than 4 shards, but this is a simple example that depicts how losing up to two regions still maintains availability of three of the five replicas of each shard.
+
+**Note:** Placement of shards in a replica do not mirror other replicas and shards are placed in a way to best load balance shards across available hosts in a single isolation group. This could mean that hosts in a region that have less hosts will do more work than those in a region with more hosts.
```

Comment:
same comment, can we only say this once in a more general section since it applies to racks too?